OptimizelySDK/Config/HttpProjectConfigManager.cs

```diff
@@ -201,12 +202,7 @@ protected override ProjectConfig Poll()
         {
             var datafile = GetRemoteDatafileResponse();
 
-            if (datafile == null)
-            {
-                return null;
-            }
-
-            return DatafileProjectConfig.Create(datafile, Logger, ErrorHandler);
```

Comment:
Thanks for the question @Mat001.  This was a refactor for clarity ... but perhaps it's not clear.  I'm moving it back.